skimage/restoration/deconvolution.py

```diff
@@ -387,6 +387,7 @@ def richardson_lucy(image, psf, iterations=50, clip=True):
 
     for _ in range(iterations):
         relative_blur = image / convolve_method(im_deconv, psf, 'same')
+        relative_blur[np.isnan(relative_blur)] = 0  # to prevent NaN from propogating throughout convulated matrix
```

Comment:
what about `np.nan_to_num()` - https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.nan_to_num.html

Reply:
+1

Reply:
Propagating.

And, "result of convolution" instead of "convulated matrix"?